sasmodels/compare.py

```diff
@@ -872,7 +861,7 @@ def plot_profile(model_info, label='base', **args):
 
 
 def run_models(opts, verbose=False):
-    # type: (Dict[str, Any]) -> Dict[str, Any]
+    # type: (Dict[str, Any], bool) -> Dict[str, Any]
```

Comment:
Optional[bool]?

Reply:
No. `Optional[bool]` would accept `None` as input. This is `bool` with a default value specified.